### ISSUE TITLE
Global sample ratio overrides route-specific

### ISSUE THREAD

**Issue by burdikov** [Kong/kong-plugin-zipkin#106]
Hello, guys,

Title says it all pretty much. Not sure if this issue is with the plugin (since the config is calculated by kong), but other plugins seem to prioritize configuration values correctly.

So, having config like this
```
_format_version: "2.1"
_transform: true

services:
- name: echo
  url: 'https://httpbin.org'
  routes:
  - name: echo
    paths:
    - /echo
    plugins:
    - name: zipkin
      config:
        "http_endpoint": "http://jaeger-collector:9411/api/v2/spans"
        "sample_ratio": 1

plugins:
- name: zipkin
  config:
    "http_endpoint": "http://jaeger-collector:9411/api/v2/spans"
    "sample_ratio": 0
```
will leave without any traces from requests to  `/echo`.

**Comment by kikito** [comment]
Thanks for writing such a detailed issue, we'll try to investigate it as soon as possible.

**Comment by kikito** [comment]
Hello @burdikov,

I programmed a fix for this problem and just released version 1.4.1 of the plugin with it on it.

**Comment by burdikov** [comment]
Thanks a lot!
